config/vufind/config.ini

```diff
@@ -1721,6 +1721,12 @@ jump_to_single_search_result = false
 ; punctuation, but this can be used when ISBD punctuation is absent (e.g. ", ").
 ;marcPublicationInfoSeparator = " "
 
+; If you have a custom index, you might want to change the field
+; where the full MARC record is supposed to be.
+; You can do that with this configuration setting.
```

Comment:
It might be worth adding a comment here to note that the preferred field will only be used if it is present in the index, and otherwise fullrecord will be used.

Reply:
Agreed. I will incorporate this in the next version.